src/components/tab-nav/tab-nav.scss

```diff
@@ -50,11 +50,21 @@
   @apply top-0;
 }
 
+:host([position="bottom"]) .tab-nav-active-indicator {
+  bottom: unset;
```

Comment:
Do you know if we could construct the CSS so we don't have to use "unset"?

Reply:
as discussed in teams, the bottom needs to be unset only in two scenarios and in all other cases it is required to have `bottom-0`.